components/footer.tsx

```diff
@@ -16,13 +16,14 @@ const Footer = () => {
         <div className="flex flex-col justify-between gap-x-2 gap-y-8 px-8 md:flex-row">
           <div className="flex flex-col md:gap-2">
             <Link href="/" className="w-fit">
-              <Image
-                src={"/codedevils-logo.svg"}
-                alt="CodeDevils logo"
-                width={150}
-                height={150}
-                className=""
-              />
+            <div className="w-[150px] h-[150px] relative">
+                <Image
+                  src="/codedevils-logo.svg"
+                  alt="CodeDevils logo"
+                  fill
```

Comment:
The fill property should only be used when the [width](https://nextjs.org/docs/pages/api-reference/components/image#width) and [height](https://nextjs.org/docs/pages/api-reference/components/image#height) of an image are unknown. In this case, please ensure you're using the width and height properties instead.